src/vllm_tgis_adapter/grpc/grpc_server.py

```diff
@@ -861,18 +856,25 @@ async def Tokenize(
         tokenizer = await self._get_tokenizer(adapter_kwargs)
 
         responses: list[TokenizeResponse] = []
+        is_mistral_tokenizer = isinstance(tokenizer, MistralTokenizer)
 
         # TODO: maybe parallelize, also move convert_ids_to_tokens into the
         # other threads
         for req in request.requests:
-            batch_encoding = tokenizer.encode_plus(
-                text=req.text,
-                return_offsets_mapping=request.return_offsets,
-                add_special_tokens=ADD_SPECIAL_TOKENS,
-            )
+            if is_mistral_tokenizer:
```

Comment:
I guess I could also check if the tokenizer has the `encode_plus` attribute to keep things generic

Reply:
@prashantgupta24 yes this would probably be better, but I think this is ok too.

Reply:
Doing this now

Reply:
Done